### ISSUE TITLE
not receiving messages posted by others, matrix room

### ISSUE THREAD

**Issue by Hanskes** [nextcloud/talk_matterbridge#42]
I have set up a matrix bridging channel in Talk to a matrix room. When I post a message in Talk it appears in the matrix room. If someone else posts a message in the matrix room, it will not appear in Talk. Is there anything I should still set? In Talk or in the matrix room?
 
![matrix](https://user-images.githubusercontent.com/33070480/110943207-3aef4300-833b-11eb-991f-99c694fa9ad6.png)

the log file:

 time="2021-03-12T12:45:35Z" level=info msg="Running version 1.22.0 afcd362c" prefix=main
time="2021-03-12T12:45:35Z" level=info msg="Parsing gateway myGateway" prefix=router
time="2021-03-12T12:45:35Z" level=info msg="Starting bridge: matrix.0 " prefix=router
time="2021-03-12T12:45:35Z" level=info msg="Connecting https://matrix-client.matrix.org" prefix=matrix
time="2021-03-12T12:45:36Z" level=info msg="Connection succeeded" prefix=matrix
time="2021-03-12T12:45:36Z" level=info msg="matrix.0: joining #nietvergeten:matrix.org (ID: #nietvergeten:matrix.orgmatrix.0)" prefix=matrix
time="2021-03-12T12:45:36Z" level=info msg="Starting bridge: nctalk.1 " prefix=router
time="2021-03-12T12:45:36Z" level=info msg=Connecting prefix=nctalk
time="2021-03-12T12:45:36Z" level=info msg=Connected prefix=nctalk
time="2021-03-12T12:45:36Z" level=info msg="nctalk.1: joining esx5wr5a (ID: esx5wr5anctalk.1)" prefix=nctalk
time="2021-03-12T12:45:36Z" level=info msg="Gateway(s) started succesfully. Now relaying messages" prefix=main

**Comment by maste9** [comment]
Same problem here. Log says gateway`s working, receiving every message in the Matrix-Room but no reaction if I answer there with another account. So the communication is basically one-way?
The Matrix-Room is encrypted - does that make any difference?

**Comment by gary-kim** [comment]
Yes, Matterbridge does not support encrypted Matrix rooms. There's a ticket open for it here: https://github.com/42wim/matterbridge/issues/1187

**Comment by maste9** [comment]
Thank you for making that clear. It doesn't seem to be fixed anytime soon according to the comment dates of the issue.
